README.md

```diff
@@ -0,0 +1,48 @@
+# Segment LDK
+
+This wrapper library makes use of [Segment](https://segment.com/docs/) to send loop data to analytics services like GA and Mixpanel. It's intent is to make it easier for developers to add analytics to loops, reduce code bloat and most importantly to be consistent with event trigger names/labels. Feel free to change the types to match event trigger names specific for your use case.
+
+## How to use
+
+In the root file that starts the loop, add the following ahead of prior code.
+
+```
+import User from '../user';
+import Segment from 'loop-segment';
+
+  /*
+  * You can get the userId from user.jwt() aptitude which has been abstracted to a
+  * separate class User
+  */
+  const { userId } = await User.getUser();
```

Comment:
I don't know how I feel about this. As the user of the library could be using a different authentication/user management system. 

I would just update comment not necessarily the code to be something like 
```
We are using the user aptitude to retrieve our user identifier but this could be populated with user id populated from other sources. 
```

There is a probably a better way to say that but our user ids wont make any sense or have any context to them unless they have a mapping to the sub that gets returned to a 3rd part dev

Reply:
Good call, will make that more clear